common/Makefile

```diff
@@ -22,7 +22,7 @@ clean:
 
 oshinko-cli:
 	- mkdir -p $(clonepath)
-	- cd $(clonepath) && git clone -b v0.2.1 https://github.com/radanalyticsio/oshinko-cli
+	- cd $(clonepath) && git clone -b ephemeral https://github.com/tmckayus/oshinko-cli
```

Comment:
Note that currently this PR pulls the oshinko-cli from a private branch. This is because the corresponding changes to the CLI have not yet been merged to master (PR pending).